### ISSUE TITLE
Delete Database: Controls of 'Delete Database' blade are not visible at 200% resize mode.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1045]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1242667

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate through all the controls under SQL API Section.
 Navigate to "Delete Data Base" menu item from the more options(...) of 'Sample Database' and select it.
Navigate through all the controls under "Delete Data Base" blade.
Resize the web page up to 200% mode.
Verify the issue.

Actual Result:
Controls of 'Delete Database' blade like edit fields are not visible at 200% resize mode.

Expected Result:
Controls of 'Delete Database' blade must be visible completely and properly at 200% resize mode..

Note:
Only 'Warning message' and 'OK' button is visible at 200% resize mode.
This issue exists throughout the application.

![image](https://user-images.githubusercontent.com/8519407/132717110-1041c489-20df-407f-bded-d6e94106588a.png)
![image](https://user-images.githubusercontent.com/8519407/132717137-ae4fc276-fc25-46c9-9469-da9f2a058105.png)
